recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set version = "0.7.6" %}
+{% set version = "0.7.7" %}
 
 package:
   name: drmaa
   version: {{ version }}
 
 source:
   fn: drmaa-{{ version }}.tar.gz
-  url: https://pypi.python.org/packages/source/d/drmaa/drmaa-{{ version }}.tar.gz
-  md5: 3cb6dade683d0621fb14871ef1261f9b
+  url: https://pypi.io/packages/source/d/drmaa/drmaa-{{ version }}.tar.gz
+  md5: 2a252032b9b1b49b7f93bc38b987f6df
```

Comment:
Would add that you can use `sha256`.

ref: https://pypi.org/project/drmaa/0.7.7/#files